torch/ao/sparsity/utils.py

```diff
@@ -0,0 +1,20 @@
+from torch import nn
+
+# Parametrizations
+class FakeSparsity(nn.Module):
+    r"""Parametrization for the weights. Should be attached to the 'weight' or
+    any other parmeter that requires a mask applied to it.
+
+    Note::
+
+        Once the mask is passed, the variable should not change the id. The
+        contensts of the mask can change, but the mask reference itself should
+        not.
+    """
+    def __init__(self, mask):
+        super().__init__()
```

Comment:
Should we instead init with shape instead of mask?
The mask could be initialized to torch.ones(shape)
``` python
  def __init__(self, shape):
      super().__init__()
      self.register_buffer('mask', torch.ones(shape))
```

If the user wants to override the mask, they can directly set the attribute.

Reply:
What if the user already has a mask?